web/js/07_grid.js

```diff
@@ -43,6 +44,8 @@ Grid.prototype.setCurrentTab = function (idx) {
   this.tabs[this.currentTabIdx].style.display = 'none'
   this.tabs[idx].style.display = 'block'
   this.currentTabIdx = idx
+  for (s of this.switchers)
+    s.value = idx
   for (w of this.widgets)
     w.applyState(w.getState())
```

Comment:
Přijde mi lepší nemít zvlášť pole se switchery, ale v tom spodním foru udělat `if(w instanceof Switcher)  { ... }`, stejně se to projíží všechno.